src/components/SendComment.vue

```diff
@@ -53,7 +52,7 @@ export default {
         (data) => client.signMessage(data),
         this.parentId,
       );
-      EventBus.$emit('reloadData');
+      this.$store.commit('reloading', true);
```

Comment:
with committing to the store to me its not implicit whats happening, emitting some event is way more clear

Reply:
What is your proposal? Should we leave the old implementation?

Reply:
my proposal is to remove the event bus, but not by emulating an event bus with vuex, but use vuex as intended or imagined (cc @davidyuk )